sdk/src/test/java/io/dapr/it/state/HelloWorldClientIT.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.it.state;
+
+import io.dapr.DaprGrpc;
+import io.dapr.DaprProtos;
+import io.dapr.it.DaprIntegrationTestingRunner;
+import io.dapr.it.services.EmptyService;
+import io.dapr.it.services.HelloWorldGrpcStateService;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.junit.*;
+
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+
+public class HelloWorldClientIT {
+
+
+    private static DaprIntegrationTestingRunner daprIntegrationTestingRunner;
+
+    @BeforeClass
+    public static void init() throws IOException, InterruptedException, TimeoutException, ExecutionException {
+         daprIntegrationTestingRunner =
+                DaprIntegrationTestingRunner.createDaprIntegrationTestingRunner(
+                        "BUILD SUCCESS",
+                        HelloWorldGrpcStateService.class,
+                        true,
+                        50002,
+                        2000
+                        );
+        daprIntegrationTestingRunner.initializeDapr();
+    }
+
+    @Test
+    public void testHelloWorldState(){
+        ManagedChannel channel =
+                ManagedChannelBuilder.forAddress("localhost", 50002).usePlaintext().build();
```

Comment:
Here the env variable will not be set. So, we need some code to find a free port,

Reply:
done